.github/workflows/deploy-staging.yaml

```diff
@@ -0,0 +1,24 @@
+name: Deploy (Staging)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - "devops-fix"
+    - "master"
```

Comment:
```suggestion

```
Let's remove this for now as it's still quite volatile after the migration. Wouldn't want to deploy any accidental push